domain/src/DomainForm.php

```diff
@@ -4,11 +4,65 @@
 
 use Drupal\Core\Entity\EntityForm;
 use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\domain\DomainLoader;
+use Drupal\domain\DomainValidator;
+use Drupal\domain\DomainCreator;
+use Drupal\Core\Path\PathValidatorInterface;
 
 /**
  * Base form for domain edit forms.
  */
-class DomainForm extends EntityForm {
+class DomainForm extends EntityForm implements ContainerInjectionInterface {
```

Comment:
I didn't know how to do this. If nothing else, this part of the patch should get in.